stylesheets/components/modal.css

```diff
@@ -17,10 +17,13 @@
   padding: 1rem;
   max-width: 100%;
   flex-direction: column;
+  min-width: 15rem;
 }
```

Comment:
I can't see a display:flex on modal, curious how flex-direction is working then. In case flex is a utility class you've used, you can do the same for flex direction: column, something like 
.flex-col{
flex-direction: colum;
}
And then use that across your library.